pkgs/applications/science/biology/manta/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, cmake, zlib, python2 }:
+
+stdenv.mkDerivation rec {
+  pname = "manta";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "Illumina";
+    repo = "manta";
+    rev = "v${version}";
+    sha256 = "1711xkcw8rpw9xv3bbm7v1aryjz4r341rkq5255192dg38sgq7w2";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ zlib python2 ];
+  postFixup = ''
+    sed -i 's|/usr/bin/env python2|${python2}/bin/python|' $out/lib/python/makeRunScript.py
+    sed -i 's|/usr/bin/env python|${python2}/bin/python|' $out/lib/python/pyflow/pyflow.py
+    sed -i 's|/bin/bash|${stdenv.shell}|' $out/lib/python/pyflow/pyflowTaskWrapper.py
+  '';
+  doInstallCheck = true;
+  installCheckPhase = ''
+    rm $out/lib/python/**/*.pyc
+    PYTHONPATH=$out/lib/python:$PYTHONPATH python -c 'import makeRunScript'
```

Comment:
`$out/lib/python` is not a typical site-packages folder. Why does it not install into `python2.sitePackages`?

Reply:
Manta is not a python package, it is a c++ software with some supporting python code. They use python to manage the execution of the c++ core, i.e., the workflow, to allow computation on various types of computational resources (slurm/pbs clusters, cloud, direct, etc). The process is complicated and documented [here](https://github.com/Illumina/manta/blob/master/docs/userGuide/README.md#run-configuration-and-execution). I've verified complete execution on a non-trivial example.

Reply:
the python scripts are more for utility, than being consumed by other python packages

Reply:
Is this resolved?

Reply:
Personally, I would say so